src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -21,7 +21,7 @@ namespace processor {
 ParsingDriver::ParsingDriver(common::DataChunk& chunk) : chunk(chunk), rowEmpty(false) {}
 
 void copyStringToVector(ValueVector* vector, uint64_t rowToAdd, std::string_view strVal,
```

Comment:
I think curType is only used for fixedList right?
To make the interface and implementation cleaner, you can use var-size template and implement specialization for each dataType.